code/game/objects/items/shards.dm

```diff
@@ -76,8 +76,13 @@
 	var/mob/living/M = AM
 	if(M.status_flags & INCORPOREAL)
 		return ..()
-
-	playsound(src.loc, 'sound/effects/glass_step.ogg', 25, 1) // not sure how to handle metal shards with sounds
+	
+	if(prob(20))
+		playsound(src.loc, 'sound/weapons/genhit.ogg', 25, TRUE)
+		qdel(src)
+	else
+		playsound(src.loc, 'sound/effects/glass_step.ogg', 25, TRUE) // not sure how to handle metal shards with sounds
+	
```

Comment:
```suggestion
	playsound(loc, 'sound/effects/glass_step.ogg', 25, TRUE)
	if(prob(20))
	    to_chat(M, "<span class='danger'>[isxeno(M) ? "We" : "You"] step on \the [src], shattering it!</span>")
		qdel(src)
		return 
	
```

After playing around with this locally, i think you'd be better just having a single sound, and add a message to say you broke it.

The gen hit sound didn't really sound great.

Reply:
yeah I was trying to find a crunchy sound but not many options